yaml/csi-driver/edge/hpe-csi-k8s-1.20.yaml

```diff
@@ -796,7 +796,7 @@ spec:
             - name: registration-dir
               mountPath: /registration
         - name: hpe-csi-driver
-          image: quay.io/hpestorage/csi-driver:v1.5.0-beta
+          image: quay.io/hpestorage/csi-driver:v2.0.0-beta
```

Comment:
Do we not need to update the /var/lib/kubelet in all the edge yamls, if user's kubelet root dir is different than default?

Reply:
I was thinking that we support this only via helm since it affects several kubelet path in node plugin.